sys-utils/eject.c

```diff
@@ -911,8 +911,8 @@ int main(int argc, char **argv)
 		verbose(&ctl, _("%s: is whole-disk device"), ctl.device);
 	}
 
-	if (ctl.F_option == 0 && is_hotpluggable(&ctl) == 0)
-		errx(EXIT_FAILURE, _("%s: is not hot-pluggable device"), ctl.device);
+	if (ctl.F_option == 0 && is_removable(&ctl) == 0)
+		errx(EXIT_FAILURE, _("%s: is not ejectable device"), ctl.device);
```

Comment:
Maybe use "eject-able", but I'm the last one who can recommend anything related to English ;-)

Reply:
"ejectable" seems to be correct: https://www.merriam-webster.com/dictionary/ejectable